gateway-ha/src/test/java/io/trino/gateway/ha/router/TestTrinoQueryProperties.java

```diff
@@ -56,4 +57,38 @@ void testJsonCreator()
         assertThat(deserializedTrinoQueryProperties.isQueryParsingSuccessful()).isEqualTo(trinoQueryProperties.isQueryParsingSuccessful());
         assertThat(deserializedTrinoQueryProperties.getErrorMessage()).isEqualTo(trinoQueryProperties.getErrorMessage());
     }
+
+    @Test
+    void testJsonCreatorWithEmptyProperties()
```

Comment:
I guess code readers won't be able to understand what's empty because some fields are present. Please rename this test method.